governance/handler.go

```diff
@@ -440,3 +440,22 @@ func (gov *Governance) GetGovernanceItemAtNumber(num uint64, key string) (interf
 func (gov *Governance) GetItemAtNumberByIntKey(num uint64, key int) (interface{}, error) {
 	return gov.GetGovernanceItemAtNumber(num, GovernanceKeyMapReverse[key])
 }
+
+// GetGoverningNodeAtNumber returns whether the governing mode is single or not and the governing node.
+func (gov *Governance) GetGoverningNodeAtNumber(num uint64) (bool, common.Address, error) {
```

Comment:
This function returns two kinds of information. What about changing the name of the function to contain both information?

Reply:
@yoomee1313 Agreed. I updated the method name, but please let me know if you have any suggestions.